src/segment_language.go

```diff
@@ -33,6 +43,10 @@ const (
 	MissingCommandTextProperty Property = "missing_command_text"
 	// MissingCommandText displays empty string by default
 	MissingCommandText string = ""
+	// VersionRegex sets the regex used to detect the language version
+	VersionRegex string = "version_regex"
```

Comment:
I would not expose this. Most users won't even know what to do if this fails and we're faster anyways to create a quick fix.

Reply:
Makes sense. Was added for more flexibility but in the end, probably 90% of users just want it to work.

Reply:
version_regex and url_template removed from config. As you said, it's better to keep it internal and update it if an issue arise.